website/src/components/views/overview/LinkedOverview.astro

```diff
@@ -0,0 +1,80 @@
+};
+
+const { organism, linkedOrganismsTitle, linkedOrganisms } = Astro.props;
+
+const section = getPathogenMegaMenuSections()[organism];
+
+const allSections = getPathogenMegaMenuSections();
+const subsections = linkedOrganisms.map((key) => allSections[key]);
+---
+
+<ContaineredPageLayout
+    title={section.headline}
+    breadcrumbs={[...defaultBreadcrumbs, { name: section.headline, href: section.href }]}
+>
+    <PageHeadline>{section.headline}</PageHeadline>
+    <div class='group flex flex-wrap justify-center gap-2'>
+        {
+            section.navigationEntries.map((entry) => {
+                return (
+                    <div class={`card card-border w-96 shadow ${section.borderEntryDecoration}`}>
+                        <a href={entry.href} class='card-body h-full'>
+                            <div class='card-title'>
+                                <h2 class={`iconify ${iconMapping[entry.iconType]}`} />
+                                <div>{entry.label}</div>
+                            </div>
+                            <p>{entry.description}</p>
+                            <div class='card-actions justify-end'>
+                                <span class={`btn btn-soft ${section.headlineBackgroundColor}`}>
+                                    Go to {entry.label}
+                                </span>
+                            </div>
+                        </a>
+                    </div>
+                );
+            })
+        }
+    </div>
+    <h1 class='mt-10 mb-4 text-xl font-bold'>{linkedOrganismsTitle}</h1>
+    <div class='items mx-auto mb-8 flex max-w-[1200px] flex-wrap justify-center gap-x-4 gap-y-8'>
+        {
+            Object.values(subsections).map((section) => (
+                <div class='group w-[260px]'>
+                    <div class={`${section.borderEntryDecoration} rounded-md border-2 border-gray-100`}>
+                        <h3
+                            class={`rounded-t-md ${section.headlineBackgroundColor} ${section.headlineBackgroundColorFocus}`}
+                        >
+                            <a class='block p-4 text-xl' href={section.href}>
+                                {section.headline}
+                            </a>
+                        </h3>
+                        <ul class='mx-6 mb-4'>
+                            {section.navigationEntries.map((entry) => (
+                                <MainMenuItem
+                                    iconType={entry.iconType}
+                                    menuListEntryDecoration={entry.underlineColor}
+                                    href={entry.href}
+                                    external={entry.externalLink}
+                                >
+                                    {entry.label}
+                                </MainMenuItem>
+                            ))}
+                        </ul>
+                    </div>
+                </div>
```

Comment:
Is this any different from what we use on the main page? Can we extract it into a component and reuse it here?